### PR TITLE
Signup Form File

### LINKED ISSUE #66: Add Functional Sign up Page
# Tech Stack :
- HTML
- CSS
- JavaScript

# Description
A simple attractive signup page 
IMG-20221014-WA0001

### PR DESCRIPTION
**Pull request by Nirvik07**
## Description

A simple sign up form is created and validation is done with JavaScript.

Fixes #66 

## Dependencies

N/A

## Future Improvements

N/A
## Screenshots of relevant screens

https://user-images.githubusercontent.com/97960335/195770968-cb82e430-6491-481f-859e-040f28a425ee.mp4

## Developer's checklist

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-check on my work

**If changes are made in the code:**

- [X] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [X] My changes in code generate no new warnings
- [X] My changes are breaking another fix/feature of the project
- [X] I have added test cases to show that my feature works
- [X] I have added relevant screenshots in my PR

### CODE REVIEW

**Comment by Abbhiishek** [comment]
your data is already present in `Contributors.json` file

- remove new record ( duplicate ) one 
- add your new project contribution in previous project array of object.

File: CONTRIBUTORS.json [hunk 1]
```diff
@@ -294,4 +295,34 @@
             "C"
         ]
     }
+    {
```
Comment (Abbhiishek) [inline CONTRIBUTORS.json:298]: your data is already present in `Contributors.json` file

- remove new record ( duplicate ) one 
- add your new project contribution in previous project array of object.

File: CONTRIBUTORS.json [hunk 2]
```diff
@@ -264,7 +264,8 @@
             "CSS",
             "C"
         ]
-    }, {
+    }, 
+    {
```
Comment (Abbhiishek) [inline CONTRIBUTORS.json:268]:
```json
"Projects": [{
            "id": 1,
            "name": "Webners",
            "description": "QR CODE GENERATOR",
            "link": "https://github.com/Nirvik07/Webners"
        },{
            "id": 2,
            "name": "Webners",
            "description": "SIGNUP FORM",
            "link": "https://github.com/Nirvik07/Webners"
        }],
```

**Review by Abbhiishek — CHANGES_REQUESTED** [review]
Let me know after your perform the changes

**Comment by Abbhiishek** [comment]
- This may help 😯 
```json
"Projects": [{
            "id": 1,
            "name": "Webners",
            "description": "QR CODE GENERATOR",
            "link": "https://github.com/Nirvik07/Webners"
        },{
            "id": 2,
            "name": "Webners",
            "description": "SIGNUP FORM",
            "link": "https://github.com/Nirvik07/Webners"
        }]
```

**Comment by Nirvik07** [comment]
@Abbhiishek check it.

**Pull request merged**